src/main/java/mate/academy/spring/controller/CinemaHallController.java

```diff
@@ -0,0 +1,39 @@
+package mate.academy.spring.controller;
+
+import java.util.List;
+import java.util.stream.Collectors;
+import mate.academy.spring.dto.CinemaHallRequestDto;
+import mate.academy.spring.dto.CinemaHallResponseDto;
+import mate.academy.spring.mapper.CinemaHallMapper;
+import mate.academy.spring.model.CinemaHall;
+import mate.academy.spring.service.CinemaHallService;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/cinema-halls")
+public class CinemaHallController {
+    private final CinemaHallService service;
+    private final CinemaHallMapper mapper;
+
+    public CinemaHallController(CinemaHallService service, CinemaHallMapper mapper) {
+        this.service = service;
+        this.mapper = mapper;
+    }
+
+    @GetMapping("")
+    public List<CinemaHallResponseDto> getAll() {
+        return service.getAll().stream()
+                .map(mapper::parse)
+                .collect(Collectors.toList());
+    }
+
+    @PostMapping("")
+    public CinemaHallResponseDto add(@RequestBody CinemaHallRequestDto cinemaHallRequestDto) {
+        CinemaHall cinemaHall = mapper.convert(cinemaHallRequestDto);
+        return mapper.parse(service.add(cinemaHall));
+    }
```

Comment:
```suggestion
    @GetMapping
    public List<CinemaHallResponseDto> getAll() {
        return service.getAll().stream()
                .map(mapper::parse)
                .collect(Collectors.toList());
    }

    @PostMapping
    public CinemaHallResponseDto add(@RequestBody CinemaHallRequestDto cinemaHallRequestDto) {
        CinemaHall cinemaHall = mapper.convert(cinemaHallRequestDto);
        return mapper.parse(service.add(cinemaHall));
    }
```

same in other controllers